superdesk/io/newsml_1_2.py

```diff
@@ -38,6 +38,9 @@ def parse_message(self, tree, provider):
             if parsed_el is not None:
                 item['ingest_provider_sequence'] = parsed_el.text
 
+            parsed_el = tree.find('NewsEnvelope/Priority')
+            item['Priority'] = self.map_priority(parsed_el.text if parsed_el else None)
```

Comment:
this should be `item['priority']` not `item['Priority']`. 'p' is capital, should be small.